analyses/molecular-subtyping-ATRT/results/ATRT-molecular-subtypes.tsv

```diff
@@ -1,33 +1,33 @@
-7316-3920	BS_VVVAAAXD	BS_E0S2Y0TS	NA	ATRT, SHH
-7316-878	BS_W6A0VXVR	BS_SWHQ8FDW	BS_J483EMFP	ATRT, SHH
-7316-2187	BS_WXFK57AS	BS_53TV75NN	BS_850BAHH9	ATRT, SHH
-7316-525	BS_X2FVRH45	BS_9KDA3HG6	BS_1QNTEZPS	ATRT, TYR
-7316-2224	BS_ZAQW3KMC	BS_SCQNN2A2	BS_P7EY9DXG	ATRT, SHH
-7316-2276	BS_ZDM8Y1XM	BS_0FQKT8EY	BS_996ZS2A4	ATRT, To be classified
-7316-1781	NA	BS_0EN5R1AP	BS_WVW3W6ZB	ATRT, To be classified
-7316-2090	NA	BS_19EJ85F8	NA	ATRT, To be classified
-7316-2181	NA	BS_NDSWS9TS	BS_KXCY5BN5	ATRT, To be classified
-7316-3045	NA	NA	BS_983Z6CA6	ATRT, To be classified
+Kids_First_Participant_ID	sample_id	Kids_First_Biospecimen_ID_meth	Kids_First_Biospecimen_ID_DNA	Kids_First_Biospecimen_ID_RNA	molecular_subtype
+PT_JV9G8MFD	7316-2269	BS_0Q1FFAM2	BS_ZAT0XJE5	BS_9FHSNVS0	ATRT, SHH
+PT_32J909WM	7316-2688	BS_1BDNZYMM	BS_JDMM2XG4	BS_WV0S8N4M	ATRT, To be classified
+PT_RRG1V7EP	7316-820	BS_2DPWXV8H	BS_P7CY04E7	BS_N0877CX1	ATRT, MYC
+PT_XRQHMBHB	7316-3937	BS_2XKEWVAT	BS_R5G3QP11	BS_QX12J6W1	ATRT, MYC
+PT_M04EC74B	7316-918	BS_46BKSCKR	BS_CKY2TTXG	BS_31TF0P1N	ATRT, To be classified
+PT_JCECCFPB	7316-479	BS_5V730EQB	BS_YZ0P5E8H	BS_90B22NGP	ATRT, To be classified
+PT_ESHACWF6	7316-1790	BS_5X8XV0CC	BS_A94A7EJ4	BS_MV6ASWV9	ATRT, SHH
+PT_E6FTW238	7316-3890	BS_8CN9JW34	BS_5WK2KXYW	BS_S71RJ8DH	ATRT, MYC
+PT_XTVQB9S4	7316-2186	BS_8HTKD2CD	BS_NXZSD2KK	BS_6G6C7VPX	ATRT, TYR
+PT_J302B8Y5	7316-386	BS_A5BZYXKN	BS_GTAJFTTR	BS_K25F4J3V	ATRT, SHH
+PT_CW0BJE0Y	7316-41	BS_BGJ8P34J	BS_12ZB7R6A	BS_15ETQ0E4	ATRT, MYC
+PT_ND34A6PR	7316-122	BS_BZVRE7D2	BS_QMHND3QK	BS_DHA5FMVW	ATRT, MYC
+PT_WG2Z95B5	7316-2141	BS_EXTTYXYZ	BS_8WJR4AA3	BS_MMNRZ1EQ	ATRT, MYC
+PT_MTE126WM	7316-376	BS_FARP2SQ9	BS_74A1TB03	BS_M4923M40	ATRT, MYC
+PT_XTVQB9S4	7316-2933	BS_KF5Q3K89	BS_DE26D072	BS_74FS6GMW	ATRT, SHH
+PT_RJ1TJ2KH	7316-2110	BS_M4GKP5G2	BS_6GN1FGTB	BS_4E1G7E3G	ATRT, TYR
+PT_4J24F353	7316-1137	BS_NDRBA7M5	NA	BS_GAGRGSSX	ATRT, To be classified
+PT_3KM9W8S8	7316-1771	BS_Q8NDZ7KK	BS_859AV1DB	BS_MVW01FPB	ATRT, SHH
+PT_TV5DYHNC	7316-1073	BS_RYZJYBCW	BS_W23ERYW3	BS_PGHK1JRW	ATRT, MYC
+PT_RJ1TJ2KH	7316-3680	BS_SK7VDHHN	BS_6VKKMM6M	BS_WWKNJSJ5	ATRT, To be classified
+PT_FYECBRPP	7316-1778	BS_SRXD3BH3	BS_WCNPE0AC	BS_B1HVNV0T	ATRT, SHH
+PT_7MZRW1DH	7316-2582	BS_T2DM3BW8	BS_1C4WCNQF	BS_GJNCYABP	ATRT, To be classified
+PT_6N825561	7316-3920	BS_VVVAAAXD	BS_E0S2Y0TS	NA	ATRT, SHH
+PT_ESHACWF6	7316-878	BS_W6A0VXVR	BS_SWHQ8FDW	BS_J483EMFP	ATRT, SHH
+PT_HVZTF42R	7316-2187	BS_WXFK57AS	BS_53TV75NN	BS_850BAHH9	ATRT, SHH
+PT_5CKDR2SF	7316-525	BS_X2FVRH45	BS_9KDA3HG6	BS_1QNTEZPS	ATRT, TYR
+PT_H8Z52K09	7316-2224	BS_ZAQW3KMC	BS_SCQNN2A2	BS_P7EY9DXG	ATRT, SHH
+PT_RX1EWBP9	7316-2276	BS_ZDM8Y1XM	BS_0FQKT8EY	BS_996ZS2A4	ATRT, To be classified
+NA	7316-1781	NA	BS_0EN5R1AP	BS_WVW3W6ZB	ATRT, To be classified
+NA	7316-2090	NA	BS_19EJ85F8	NA	ATRT, To be classified
+NA	7316-2181	NA	BS_NDSWS9TS	BS_KXCY5BN5	ATRT, To be classified
+NA	7316-3045	NA	NA	BS_983Z6CA6	ATRT, To be classified
```

Comment:
This seems a bit suspect

Reply:
I also noticed and am working on it. Will be fixed in next commit